vars/runSendEmail.groovy

```diff
@@ -20,11 +20,12 @@ def call(Map params, RunWrapper currentBuild){
     env
     echo "Start send email ..."
     RUNNER_IP=\$(cat sct_runner_ip||echo "")
+    EMAIL_RECIPIENTS_ARG=\$(if [[ -z "${email_recipients}" ]]; then echo ""; else echo "--email-recipients ${email_recipients}"; fi)
```

Comment:
why not just skipping send-email if no email recipients are provided?

Reply:
Do you mean skipping of the `send email` stage?
If so, then it will have side-effect on the `stage view` of each CI job.
If all stages are unchanged then we see 10 CI jobs by default.
When we change stages then we get displayed only the latest test run.

Reply:
You could just not execute the send email command if no email recipients are present, without changing the stages

Reply:
Done